src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -143,9 +143,28 @@ public static class TimeSeriesCatalog
         /// </format>
         /// </example>
         public static SrCnnAnomalyEstimator DetectAnomalyBySrCnn(this TransformsCatalog catalog, string outputColumnName, string inputColumnName,
-            int windowSize=64, int backAddWindowSize=5, int lookaheadWindowSize=5, int averageingWindowSize=3, int judgementWindowSize=21, double threshold=0.3)
+            int windowSize = 64, int backAddWindowSize = 5, int lookaheadWindowSize = 5, int averageingWindowSize = 3, int judgementWindowSize = 21, double threshold = 0.3)
             => new SrCnnAnomalyEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnName, windowSize, backAddWindowSize, lookaheadWindowSize, averageingWindowSize, judgementWindowSize, threshold, inputColumnName);
 
+        /// <summary>
+        /// Create <see cref="RootCauseLocalizationEstimator"/>, which localizes root causes using decision tree algorithm.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// The column data is an instance of <see cref="Microsoft.ML.TimeSeries.RootCause"/>.</param>
+        /// <param name="inputColumnName">Name of the input column.
+        ///  The column data is an instance of <see cref="Microsoft.ML.TimeSeries.RootCauseLocalizationInput"/>.</param>
+        /// <param name="beta">The weight parameter in score. The range of the parameter should be in [0,1].</param>
```

Comment:
Can you elaborate on the beta parameter?

Reply:
> Can you elaborate on the beta parameter in the comment?

updated